[17.02.2020 18:56:33, поток 0] [DEBUG] Start...
[17.02.2020 18:56:33, поток 0] Берём заданные куки...
[17.02.2020 18:56:34, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:56:34, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 18:56:34, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 18:57:34, поток 0] [DEBUG] Start...
[17.02.2020 18:57:34, поток 0] Берём заданные куки...
[17.02.2020 18:57:36, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:57:38, поток 0] Спарсили куки
[17.02.2020 18:57:40, поток 0] Войти в аккаунт не удалось!
[17.02.2020 18:57:41, поток 0] [DEBUG] Start...
[17.02.2020 18:57:41, поток 0] Берём заданные куки...
[17.02.2020 18:57:41, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:57:41, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 18:57:41, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 18:58:24, поток 0] [DEBUG] Start...
[17.02.2020 18:58:24, поток 0] Берём заданные куки...
[17.02.2020 18:58:25, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:58:25, поток 1] [DEBUG] Start...
[17.02.2020 18:58:25, поток 1] Берём заданные куки...
[17.02.2020 18:58:25, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 18:58:25, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 18:58:26, поток 1] Сообщение отправлено.
[17.02.2020 18:58:31, поток 1] [DEBUG] Start...
[17.02.2020 18:58:31, поток 1] Берём заданные куки...
[17.02.2020 18:58:32, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:58:33, поток 1] Спарсили куки
[17.02.2020 18:58:34, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:58:34, поток 1] [DEBUG] Start...
[17.02.2020 18:58:34, поток 1] Берём заданные куки...
[17.02.2020 18:58:35, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:58:37, поток 1] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Не удалось соединиться с прокси-сервером '95.165.151.65:47861'. ---> System.Net.Sockets.SocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 95.165.151.65:47861
   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   в xNet.ProxyClient.<>c__DisplayClass38_0.<CreateConnectionToProxy>b__0(IAsyncResult ar)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.ProxyClient.CreateConnectionToProxy()
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 18:58:37, поток 1] [DEBUG] Сменили прокси.
[17.02.2020 18:59:25, поток 0] [DEBUG] Start...
[17.02.2020 18:59:25, поток 0] Берём заданные куки...
[17.02.2020 18:59:37, поток 1] [DEBUG] Start...
[17.02.2020 18:59:37, поток 1] Берём заданные куки...
[17.02.2020 18:59:38, поток 1] Сообщение отправлено.
[17.02.2020 18:59:44, поток 1] [DEBUG] Start...
[17.02.2020 18:59:44, поток 1] Берём заданные куки...
[17.02.2020 18:59:45, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:46, поток 1] Спарсили куки
[17.02.2020 18:59:46, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:47, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:59:47, поток 1] [DEBUG] Start...
[17.02.2020 18:59:47, поток 1] Берём заданные куки...
[17.02.2020 18:59:47, поток 0] Спарсили куки
[17.02.2020 18:59:47, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:49, поток 0] Войти в аккаунт не удалось!
[17.02.2020 18:59:49, поток 1] Спарсили куки
[17.02.2020 18:59:49, поток 0] [DEBUG] Start...
[17.02.2020 18:59:49, поток 0] Берём заданные куки...
[17.02.2020 18:59:49, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:50, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 18:59:50, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 18:59:50, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:59:50, поток 1] [DEBUG] Start...
[17.02.2020 18:59:50, поток 1] Берём заданные куки...
[17.02.2020 18:59:51, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:52, поток 1] Спарсили куки
[17.02.2020 18:59:53, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:59:53, поток 1] [DEBUG] Start...
[17.02.2020 18:59:53, поток 1] Берём заданные куки...
[17.02.2020 18:59:53, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:54, поток 1] Спарсили куки
[17.02.2020 18:59:55, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:59:56, поток 1] [DEBUG] Start...
[17.02.2020 18:59:56, поток 1] Берём заданные куки...
[17.02.2020 18:59:56, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 18:59:57, поток 1] Спарсили куки
[17.02.2020 18:59:58, поток 1] Войти в аккаунт не удалось!
[17.02.2020 18:59:58, поток 1] [DEBUG] Start...
[17.02.2020 18:59:58, поток 1] Берём заданные куки...
[17.02.2020 18:59:59, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:00, поток 1] Спарсили куки
[17.02.2020 19:00:01, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:00:02, поток 1] [DEBUG] Start...
[17.02.2020 19:00:02, поток 1] Берём заданные куки...
[17.02.2020 19:00:02, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:03, поток 1] Спарсили куки
[17.02.2020 19:00:04, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:00:04, поток 1] [DEBUG] Start...
[17.02.2020 19:00:04, поток 1] Берём заданные куки...
[17.02.2020 19:00:05, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:05, поток 1] Спарсили куки
[17.02.2020 19:00:06, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:00:07, поток 1] [DEBUG] Start...
[17.02.2020 19:00:07, поток 1] Берём заданные куки...
[17.02.2020 19:00:08, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:08, поток 1] Спарсили куки
[17.02.2020 19:00:09, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:00:09, поток 1] [DEBUG] Start...
[17.02.2020 19:00:09, поток 1] Берём заданные куки...
[17.02.2020 19:00:10, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:11, поток 1] Спарсили куки
[17.02.2020 19:00:12, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:00:50, поток 0] [DEBUG] Start...
[17.02.2020 19:00:50, поток 0] Берём заданные куки...
[17.02.2020 19:00:50, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:51, поток 0] Спарсили куки
[17.02.2020 19:00:53, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:00:53, поток 0] [DEBUG] Start...
[17.02.2020 19:00:53, поток 0] Берём заданные куки...
[17.02.2020 19:00:53, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:00:54, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:00:54, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:01:54, поток 0] [DEBUG] Start...
[17.02.2020 19:01:54, поток 0] Берём заданные куки...
[17.02.2020 19:02:04, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:02:04, поток 0] Спарсили куки
[17.02.2020 19:02:06, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:02:06, поток 0] [DEBUG] Start...
[17.02.2020 19:02:06, поток 0] Берём заданные куки...
[17.02.2020 19:02:06, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:02:07, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:02:07, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:03:07, поток 0] [DEBUG] Start...
[17.02.2020 19:03:07, поток 0] Берём заданные куки...
[17.02.2020 19:03:08, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:03:08, поток 0] Спарсили куки
[17.02.2020 19:03:10, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:03:11, поток 0] [DEBUG] Start...
[17.02.2020 19:03:11, поток 0] Берём заданные куки...
[17.02.2020 19:03:11, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:03:11, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:03:11, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:04:11, поток 0] [DEBUG] Start...
[17.02.2020 19:04:11, поток 0] Берём заданные куки...
[17.02.2020 19:04:12, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:04:13, поток 0] Спарсили куки
[17.02.2020 19:04:14, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:04:15, поток 0] [DEBUG] Start...
[17.02.2020 19:04:15, поток 0] Берём заданные куки...
[17.02.2020 19:04:15, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:04:15, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:04:15, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:05:15, поток 0] [DEBUG] Start...
[17.02.2020 19:05:15, поток 0] Берём заданные куки...
[17.02.2020 19:05:16, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:05:17, поток 0] Спарсили куки
[17.02.2020 19:05:19, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:05:19, поток 0] [DEBUG] Start...
[17.02.2020 19:05:19, поток 0] Берём заданные куки...
[17.02.2020 19:05:20, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:05:20, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:05:20, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:06:20, поток 0] [DEBUG] Start...
[17.02.2020 19:06:20, поток 0] Берём заданные куки...
[17.02.2020 19:06:24, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:06:25, поток 0] Спарсили куки
[17.02.2020 19:06:26, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:06:27, поток 0] [DEBUG] Start...
[17.02.2020 19:06:27, поток 0] Берём заданные куки...
[17.02.2020 19:06:27, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:06:27, поток 0] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Получен неверный код статуса 'ServiceUnavailable' от прокси-сервера '194.183.168.4:20567'.
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:06:27, поток 0] [DEBUG] Сменили прокси.
[17.02.2020 19:07:27, поток 0] [DEBUG] Start...
[17.02.2020 19:07:27, поток 0] Берём заданные куки...
[17.02.2020 19:07:28, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:07:29, поток 0] Спарсили куки
[17.02.2020 19:07:31, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:24:44, поток 0] [DEBUG] Start...
[17.02.2020 19:24:44, поток 0] Берём заданные куки...
[17.02.2020 19:24:45, поток 1] [DEBUG] Start...
[17.02.2020 19:24:45, поток 1] Берём заданные куки...
[17.02.2020 19:24:45, поток 2] [DEBUG] Start...
[17.02.2020 19:24:45, поток 2] Берём заданные куки...
[17.02.2020 19:24:45, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:45, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:46, поток 2] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:46, поток 0] Спарсили куки
[17.02.2020 19:24:47, поток 2] Спарсили куки
[17.02.2020 19:24:48, поток 1] Спарсили куки
[17.02.2020 19:24:48, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:24:48, поток 0] [DEBUG] Start...
[17.02.2020 19:24:48, поток 0] Берём заданные куки...
[17.02.2020 19:24:48, поток 2] Войти в аккаунт не удалось!
[17.02.2020 19:24:48, поток 2] [DEBUG] Start...
[17.02.2020 19:24:48, поток 2] Берём заданные куки...
[17.02.2020 19:24:48, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:49, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:24:49, поток 1] [DEBUG] Start...
[17.02.2020 19:24:49, поток 1] Берём заданные куки...
[17.02.2020 19:24:49, поток 2] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:49, поток 0] Спарсили куки
[17.02.2020 19:24:50, поток 2] Спарсили куки
[17.02.2020 19:24:51, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:51, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:24:52, поток 0] [DEBUG] Start...
[17.02.2020 19:24:52, поток 0] Берём заданные куки...
[17.02.2020 19:24:52, поток 2] Войти в аккаунт не удалось!
[17.02.2020 19:24:52, поток 1] Спарсили куки
[17.02.2020 19:24:53, поток 2] [DEBUG] Start...
[17.02.2020 19:24:53, поток 2] Берём заданные куки...
[17.02.2020 19:24:53, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:53, поток 2] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:54, поток 0] Спарсили куки
[17.02.2020 19:24:54, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:24:54, поток 1] [DEBUG] Start...
[17.02.2020 19:24:54, поток 1] Берём заданные куки...
[17.02.2020 19:24:54, поток 2] Спарсили куки
[17.02.2020 19:24:55, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:55, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:24:55, поток 0] [DEBUG] Start...
[17.02.2020 19:24:55, поток 0] Берём заданные куки...
[17.02.2020 19:24:56, поток 1] Спарсили куки
[17.02.2020 19:24:56, поток 2] Войти в аккаунт не удалось!
[17.02.2020 19:24:56, поток 2] [DEBUG] Start...
[17.02.2020 19:24:56, поток 2] Берём заданные куки...
[17.02.2020 19:24:56, поток 0] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:57, поток 2] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:57, поток 1] Войти в аккаунт не удалось!
[17.02.2020 19:24:57, поток 1] [DEBUG] Start...
[17.02.2020 19:24:57, поток 1] Берём заданные куки...
[17.02.2020 19:24:57, поток 0] Спарсили куки
[17.02.2020 19:24:57, поток 1] [Ошибка] Что-то не так. Переделываем.
[17.02.2020 19:24:58, поток 2] Спарсили куки
[17.02.2020 19:24:59, поток 0] Войти в аккаунт не удалось!
[17.02.2020 19:24:59, поток 2] Войти в аккаунт не удалось!
[17.02.2020 19:24:59, поток 0] [DEBUG] Start...
[17.02.2020 19:24:59, поток 0] Берём заданные куки...
[17.02.2020 19:25:00, поток 2] [DEBUG] Start...
[17.02.2020 19:25:00, поток 2] Берём заданные куки...
[17.02.2020 19:25:00, поток 1] [DEBUG, ERROR] xNet.HttpException: Не удалось соединиться с HTTP-сервером 'www.instagram.com'. ---> xNet.ProxyException: Не удалось соединиться с прокси-сервером '188.32.135.96:7844'. ---> System.Net.Sockets.SocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 188.32.135.96:7844
   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   в xNet.ProxyClient.<>c__DisplayClass38_0.<CreateConnectionToProxy>b__0(IAsyncResult ar)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.ProxyClient.CreateConnectionToProxy()
   в xNet.HttpProxyClient.CreateConnection(String destinationHost, Int32 destinationPort, TcpClient tcpClient)
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   --- Конец трассировки внутреннего стека исключений ---
   в xNet.HttpRequest.CreateTcpConnection(String host, Int32 port)
   в xNet.HttpRequest.CreateConnection(Uri address)
   в xNet.HttpRequest.TryCreateConnectionOrUseExisting(Uri address, Uri previousAddress)
   в xNet.HttpRequest.Request(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, Uri address, HttpContent content)
   в xNet.HttpRequest.Raw(HttpMethod method, String address, HttpContent content)
   в xNet.HttpRequest.Get(String address, RequestParams urlParams)
   в InstaImap.Instagram.GetCookie() в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 71
   в InstaImap.Instagram..ctor(String Mail, String password, String id, String cookies, String DEVICEID, String ADID, String GUID, String PHONE_ID, Int32 Index, String Time, Int32 tt, String type, String text, Int32 delay) в D:\MaXiMAA\MaXiMAAOld\C#Готово\KWORK\InstaDirectMessage_ButDev\InstaDirectMessage_ButDev\Instagram.cs:строка 160
[17.02.2020 19:25:00, поток 1] [DEBUG] Сменили прокси.
